### PR TITLE
Complete missing parts in Arabic translation.

### PR DESCRIPTION
**Pull request by AAbouZaid**
Hi,
As part of open Mozilla nights, I fixed some typos and translated that missing parts in Arabic translation.

I will work on the rest later (ASAP).

Thanks.

### CODE REVIEW

File: locales/ar.ini [hunk 1]
```diff
@@ -83,11 +83,13 @@ py-dxr-extra = the intelligent source code indexing system
 py-tools-extra = various projects to facilitate easier automated testing of products
 py-webqa-extra = create automated tests for Mozilla's websites
 py-releng-extra = Mozilla's build, test, and release pipeline
-php-intro = So you like your variable names to include dollar signs? That's cool, everyone misses Perl once in a while. You can work on
-php-wordpress-extra.innerHTML = the code that runs our <a href="http://blog.mozilla.org">blogs</a>
-php-marketplace-extra.innerHTML =  the PHP client for <a href="https://marketplace.firefox.com">Marketplace</a>
-php-mediawiki-extra.innerHTML = the Bugzilla plugin for <a href="http://wiki.mozilla.org">Mozilla's MediaWiki</a>
-php-transvision-extra = the QA tool for Mozilla product localizers
+php-intro = إذن أنت تحب أسماء المتغيرات التي تستعمل علامة الدولار "$"؟ هذا رائع، الجميع يفتقد لغة بيرل من حين لآخر. يمكن العمل على
+So you like your variable names to include dollar signs? That's cool, everyone misses Perl once in a while. You can work on
```
Comment (jdm) [inline locales/ar.ini]: This looks like an accidental inclusion.

File: locales/ar.ini [hunk 2]
```diff
@@ -83,11 +83,13 @@ py-dxr-extra = the intelligent source code indexing system
...
+php-wordpress-extra.innerHTML = الشفرة التي تُشغل <a href="http://blog.mozilla.org">مدوناتنا</a>
+php-marketplace-extra.innerHTML = عميل PHP لأجل <a href="https://marketplace.firefox.com">Marketplace</a>
+the PHP client for <a href="https://marketplace.firefox.com">Marketplace</a>
```
Comment (jdm) [inline locales/ar.ini]: So does this.

File: locales/ar.ini [hunk 3]
```diff
@@ -77,39 +77,40 @@ go-intro = So you like to easily construct streaming data pipelines and love to
...
+py-firefox-extra = الكثير من الشفرة البرمجية تُنتج عن طريق سكربتات بايثون
+py-webdev-extra = لدينا العديد من المشاريع الكبيرة والمعقدة مكتوبة بلغة بايثون
+py-dxr-extra = الشفرة المصدرية الذكية لفهرسة النظام
+py-tools-extra = مجموعة مشاريع متنوعة لتسهيل عميلة أتمتة اختبار المنتجات
+py-webqa-extra = إنشاء إختبارات مؤتمة لمواقع موزيلا
+py-releng-extra = خط البناء، التجربة، وإلاصدار لموزيلا
+php-intro = إذن أنت تحب أسماء المتغيرات التي تستعمل علامة الدولار "$"؟ هذا رائع، الجميع يفتقد لغة بيرل من حين لآخر. يمكن العمل على
+php-wordpress-extra.innerHTML = الشفرة التي تُشغل <a href="http://blog.mozilla.org">مدوناتنا</a>
+php-marketplace-extra.innerHTML = عميل PHP لأجل <a href="https://marketplace.firefox.com">Marketplace</a>
+the PHP client for <a href="https://marketplace.firefox.com">Marketplace</a>
```
Comment (jdm) [inline locales/ar.ini]: This looks like an accidental change.

**Comment by jdm** [comment]
This is fine to merge after that extra line is removed!

**Comment by AAbouZaid** [comment]
Akh, I thought I fixed it :‑)
It has been fixed now.

**Comment by jdm** [comment]
Thanks!

**Pull request merged**